qiita_pet/support_files/doc/source/faq.rst

```diff
@@ -4,26 +4,25 @@ Frequently Asked Questions
 What kind of data can I upload to Qiita for processing?
 -------------------------------------------------------
 
-We need 3 things: raw data, sample template, and prep template. At this
-moment, raw data is fastq files without demultiplexing with forward,
-reverse (optional) and barcode reads. We should have before the end of
-the week SFF processing so it's OK to upload. Note that we are accepting
-any kind of target gene (16S, 18S, ITS, whatever) as long as they have
-some kind of demultiplexing strategy and that you can also upload WGS.
-However, WGS processing is not ready.
+Processing in Qiita requires 3 things: raw data, sample and prep information
+files. `Here <https://github.com/biocore/qiita/blob/master/README.rst#accepted-raw-files>__`
+a list of currently supported raw files files. Note that we are accepting
+any kind of target gene (16S, 18S, ITS, whatever) as long as there is
+some kind of demultiplexing strategy. You can also upload WGS however, WGS
```

Comment:
Hmmm the last part is a bit confusing " some kind of demultiplexing strategy" as before you mention that per sample sequence files are acceptable.